app/src/main/java/com/obsidian/plugins/task_notifier/os/FIleObserverService.kt

```diff
@@ -30,6 +36,8 @@ class FileObserverService : Service() {
             Logger.info("FileObserverService missing path from FILE_PATH_INTENT_KEY intent key")
             return STOP_FOREGROUND_REMOVE;
         }
+        createForeGroundNotification(context);
```

Comment:
it would be better to move the content of the createForeGroundNotification function to NotificationManager.createAndShowNotification() and here only use the startForeground(notification)